recipes/heliano/build.sh

```
@@ -0,0 +1,48 @@
+#!/bin/bash
+#Based on https://github.com/TobyBaril/EarlGrey/blob/main/configure
+set -x
+
+# Define paths
+PACKAGE_HOME=${PREFIX}/share/${PKG_NAME}-${PKG_VERSION}-${PKG_BUILDNUM}
+SCRIT_DIR_PATH="${PACKAGE_HOME}/"
```

Shellcheck

[warning] SCRIT_DIR_PATH appears unused. Verify use (or export if used externally).

(SC2034)